server/php/UploadHandler.php

```diff
@@ -795,25 +796,26 @@ protected function gd_create_scaled_image($file_name, $version, $options) {
         // Handle transparency in GIF and PNG images:
         switch ($type) {
             case 'gif':
-            case 'png':
                 imagecolortransparent($new_img, imagecolorallocate($new_img, 0, 0, 0));
+                break;
             case 'png':
+                imagecolortransparent($new_img, imagecolorallocate($new_img, 0, 0, 0));
```

Comment:
Good catch! 👍 
I'm surprised there were no issues with executing  `imagealphablending` and `imagesavealpha` for `GIF` images before your fix.